README.md

```diff
@@ -9,23 +9,22 @@
 [![npm](https://img.shields.io/npm/dm/levelup.svg)](https://www.npmjs.com/package/levelup)
 
   * <a href="#intro">Introduction</a>
-  * <a href="#platforms">Tested &amp; supported platforms</a>
-  * <a href="#basic">Basic usage</a>
+  * <a href="#platforms">Supported Platforms</a>
```

Comment:
I initially resisted but platforms with a capital P does look official :) Hmm

Reply:
We had `Promise Support` so I wanted to have the same format on all sections and I picked first letter capitalized since I like that style.

Reply:
Alright. Having a consistent style is what's important. I'll get used to it ;)